### PR TITLE
Push Notifications - Updated IsRegistered & Updated Dependencies

### PR DESCRIPTION
**Pull request by AndrewEQ**
- updated isRegistered to be more accurate in terms of the authorization and to improve the UX by starting with true as default because sometimes its not instant to get the value

### CODE REVIEW

File: Source/Fuse.PushNotifications/Common.uno [hunk 1]
```diff
@@ -17,9 +17,10 @@ namespace Fuse.PushNotifications
 		"java.util.List", 
 		"android.graphics.Color"
 	)]
-	[Require("Gradle.Dependency.ClassPath", "com.google.gms:google-services:4.2.0")]
+	[Require("Gradle.Dependency.ClassPath", "com.google.gms:google-services:4.3.2")]
 	[Require("Gradle.AllProjects.Repository", "maven {url 'https://maven.google.com'}")]
-	[Require("Gradle.Dependency.Implementation", "com.google.firebase:firebase-core:16.0.0")]
+	// [Require("Gradle.Dependency.Implementation", "com.google.firebase:firebase-core:16.0.0")]
```
Comment (mortend) [inline Source/Fuse.PushNotifications/Common.uno]: Remove commented-out code.

File: Source/Fuse.PushNotifications/iOS/Impl.uno [hunk 2]
```diff
@@ -161,20 +161,20 @@ namespace Fuse.PushNotifications
 
 		[Foreign(Language.ObjC)]
 		internal static bool IsRegisteredForRemoteNotifications()
-		@{
-			__block bool isReg = false;
+		@{
```
Comment (mortend) [inline Source/Fuse.PushNotifications/iOS/Impl.uno]: Remove trailing whitespace.

File: Source/Fuse.PushNotifications/iOS/Impl.uno [hunk 3]
```diff
@@ -161,20 +161,20 @@ namespace Fuse.PushNotifications
...
+		@{	
+			__block bool isRegisteredForRemote = true;
 
-			__block bool isNotificationsSettingsEnabled = false;
+			__block bool isNotificationsSettingsEnabled = true;
 
 			dispatch_sync(dispatch_get_main_queue(), ^{
-
-				isReg = [[UIApplication sharedApplication] isRegisteredForRemoteNotifications];
+
```
Comment (mortend) [inline Source/Fuse.PushNotifications/iOS/Impl.uno]: Remove trailing whitespace.

File: Source/Fuse.PushNotifications/Common.uno [hunk 4]
```diff
@@ -17,9 +17,10 @@ namespace Fuse.PushNotifications
...
+	[Require("Gradle.Dependency.Implementation", "com.google.firebase:firebase-analytics:17.2.1")]
```
Comment (mortend) [inline Source/Fuse.PushNotifications/Common.uno]: Is there a v20.0.0 of this package as well, same as firebase-messaging?

Comment (AndrewEQ) [reply Source/Fuse.PushNotifications/Common.uno]: Nope: https://firebase.google.com/support/release-notes/android

Comment (mortend) [reply Source/Fuse.PushNotifications/Common.uno]: OK :)

File: Source/Fuse.PushNotifications/Android/BigPictureStyleHttp.java [hunk 5]
```diff
@@ -7,7 +7,7 @@
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 import android.os.AsyncTask;
-import android.support.v4.app.NotificationCompat;
+import androidx.core.app.NotificationCompat;
```
Comment (mortend) [inline Source/Fuse.PushNotifications/Android/BigPictureStyleHttp.java]: I see some new references to `androidx` in this PR. Maybe this should wait until we've merged #1323?

Comment (AndrewEQ) [reply Source/Fuse.PushNotifications/Android/BigPictureStyleHttp.java]: oh shucks, yeah, I forgot to remove that when I was updating the previous changes, but sure, can wait for #1323

**Pull request merged**